instrumentation/servlet/servlet-3.0/javaagent-unit-tests/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+  id("otel.java-conventions")
+}
+
+dependencies {
+  testImplementation("javax.servlet:javax.servlet-api:3.0.1")
+  testImplementation(project(":instrumentation:servlet:servlet-common:bootstrap"))
```

Comment:
hopefully can remove this dependency after injecting in unit tests and not relying on ExperimentalSnippetHolder global